### ISSUE TITLE
BinanceSocketManager.user_socket() stops picking up updates after ~ 2 hrs

### ISSUE THREAD

**Issue by mchangun** [sammchardy/python-binance#828]
**Describe the bug**
The BinanceSocketManager.user_socket() stops picking up incoming messages after about 2 hrs but no errors are raised.

**To Reproduce**
Code snippet to reproduce the behavior.  After approximately 2 hrs, changes in spot positions are no longer picked up by `_user_account_listener()` as nothing gets printed to screen.:

```
import asyncio

from decimal import Decimal as D
from binance import BinanceSocketManager
from binance.client import AsyncClient

class Portfolio:
    def __init__(self, async_client: AsyncClient = None):
        self._async_client = async_client
        self.coin_pos = []

    def update_coin_position(self, symbol: str, qty):
        self.coin_pos.append((symbol, qty))

    async def run(self, loop):
        loop.create_task(self._user_account_listener())

        while True:
            print(self.coin_pos)
            await asyncio.sleep(10)

    async def _user_account_listener(self):
        bsm = BinanceSocketManager(self._async_client)
        async with bsm.user_socket() as us:
            while True:
                msg = await us.recv()
                print(msg)

                if msg is None:
                    continue
                elif msg['e'] == 'outboundAccountPosition':
                    for balance in msg['B']:
                        self.update_coin_position(balance['a'], D(balance['f']) + D(balance['l']))


async def main():
    async_client = await AsyncClient().create(api_key=API_KEY, api_secret=API_SECRET)

    portfolio = Portfolio(async_client)
    loop.create_task(portfolio.run(loop))

if __name__ == '__main__':
    loop = asyncio.get_event_loop()
    loop.create_task(main())
    loop.run_forever()

```

**Expected behavior**
Expected the socket to keep picking up changes in user account.  If it disconnects, it should either reconnect or raise an error.

**Environment (please complete the following information):**
 - Python version: 3.9.1
 - Virtual Env: virtualenv
 - OS: MacOS 11.2.1
 - python-binance version 1.02

I'm new to `asyncio` so maybe I'm doing something wrong above?

**Comment by sammchardy** [comment]
Have resolved an issue in v1.0.6. with the keepalives not being sent for the authenticated streams

**Comment by mchangun** [comment]
Thanks @sammchardy.  I just ran the code for ~ 12 hrs and it's working great!  Would you say if there is a sensible time interval to restart my code?
